src/renuka/BankingSystem.java

```diff
@@ -0,0 +1,73 @@
+package renuka;
+
+public class BankingSystem {
+	
+	String custName;
+	int totalAmt;
+	int creditCount = 0;
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value.